extension/js/common/core/crypto/key.ts

```diff
@@ -68,9 +68,9 @@ export type Contact = {
 export interface PrvKeyInfo {
   private: string;
   longid: string;
+  fingerprints?: string[];
```

Comment:
Since every use of `fingerprints` and `emails` is banging the optional with a `!`, we should not make these optional, and then remove these bangs.

Reply:
Done